### ISSUE TITLE
Unable to load a texture - 0.6.0rc

### ISSUE THREAD

**Issue by oldschooljarvis** [Famous/engine#303]
Per @DnMllr 's request on Slack:

https://gist.github.com/oldschooljarvis/b1ad98ce661985068a83

That code works on **0.5.2** and fails on **0.6.0rc**, with the following error:

`Uncaught TypeError: Cannot read property '0' of undefined`

Fails on the second line here:

```javascript
WebGLRenderer.prototype.updateSize = function updateSize(size) {
    this.canvas.width  = size[0];
```

**Comment by alexanderGugel** [comment]
should be fixed in #291

**Comment by michaelobriena** [comment]
@oldschooljarvis a fix landed as 427adea.
Can you confirm this works on develop?

**Comment by oldschooljarvis** [comment]
@michaelobriena Confirmed working, thanks!
